securedrop-admin

```diff
@@ -46,8 +46,21 @@ def activate_venv():
 def sdconfig(args):
     """Configure SD site settings"""
     activate_venv()
-    subprocess.check_call([os.path.join(ANSIBLE_PATH,'configure')],
-                            cwd=ANSIBLE_PATH, env=os.environ.copy())
+
+    ansible_cmd = [
+        'ansible-playbook',
+        # Passing an empty inventory file to override the automatic dynamic
+        # inventory script, which fails if no site vars are configured.
+        '-i', '/dev/null',
+        os.path.join(ANSIBLE_PATH, 'securedrop-configure.yml'),
```

Comment:
[PEP8 recommends it](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas) in multiline lists:

>When trailing commas are redundant, they are often helpful when a version control system is used, when a list of values, arguments or imported items is expected to be extended over time. The pattern is to put each value (etc.) on a line by itself, always adding a trailing comma, and add the close parenthesis/bracket/brace on the next line.